code/game/objects/items/weapons/storage/storage.dm

```diff
@@ -412,20 +415,21 @@
 	W.on_enter_storage(src)
 	if(use_sound)
 		playsound(src.loc, src.use_sound, 50, 0, -5)
+	if(animated)
+		animate_parent()
 	if(user)
 		W.dropped(user)
 		if(!istype(W, /obj/item/forensics))
 			add_fingerprint(user)
 
 		if(!prevent_warning)
 			for(var/mob/M in viewers(user, null))
-				if (M == usr)
-					to_chat(usr, "<span class='notice'>You put \the [W] into [src].</span>")
+				if(M == usr)
+					continue
 				else if (M in range(1)) //If someone is standing close enough, they can tell what it is...
-					M.show_message("<span class='notice'>\The [user] puts [W] into [src].</span>")
+					M.show_message(SPAN_NOTICE("\The [user] puts [W] into [src]."))
```

Comment:
```suggestion
					M.show_message(SPAN_NOTICE("\The [user] puts [W] into [src]."))
```
This feels like it should be a visible message sent to everyone. Then we can have blind and self-checking.